src/apps/socket/raw.lua

```diff
@@ -31,6 +31,11 @@ function RawSocket:push ()
    end
 end
 
+function RawSocket:close ()
```

Comment:
This should be `RawSocket:stop`. See the App API reference which lists the “special” methods: https://github.com/SnabbCo/snabbswitch/tree/v2015.10/src#app